layouter/static/layouter/css/layouter.css

```diff
@@ -70,4 +70,12 @@
 .full-width:before {
   content: "\e906";
 }
-
+@media only screen and (max-width: 480px) {
```

Comment:
@SteinRobert what do you think would be the best default media queries for the three devices?